mtags/src/main/scala-2/scala/meta/internal/pc/completions/ScaladocCompletions.scala

```diff
@@ -81,7 +90,9 @@ trait ScaladocCompletions { this: MetalsGlobal =>
             editRange,
             newText
           ),
-          completionsSymbol(associatedDef.name.toString()),
+          completionsSymbol(
+            associatedDef.map(d => d.name.toString()).getOrElse("Scaladoc")
```

Comment:
As long as I know, [sym](https://github.com/scalameta/metals/blob/5d32828171f202402088265e90066dd34fa49567/mtags/src/main/scala-2/scala/meta/internal/pc/completions/Completions.scala#L39) is used only for ordering the completion members (based on its attributes) here: https://github.com/scalameta/metals/blob/5d32828171f202402088265e90066dd34fa49567/mtags/src/main/scala-2/scala/meta/internal/pc/completions/Completions.scala#L152

Since the auto-completion for scaladoc always returns only one member, it's fine to give an arbitrary symbol to `sym` for `TextEditMember`.